src/components/tags.js

```diff
@@ -19,7 +19,8 @@ AFRAME.registerComponent("tags", {
     isHoverMenuChild: { default: false },
     isStatic: { default: false },
     inspectable: { default: false },
-    preventAudioBoost: { default: false }
+    preventAudioBoost: { default: false },
+    ignoreSpaceBubble: { default: false }
```

Comment:
Probably warants thinking through the product implications, so maybe we don't need it in this PR, but wonder if we should expose this to gltfs like

```js
AFRAME.GLTFModelPlus.registerComponent("ignore-space-bubble", "tags", el => {
  el.setAttribute("tags", { ignoreSpaceBubble: true });
});
```

So people could make things in their avatars that ignore the space bubble effect.

Reply:
I think this could be abused and we probably want to keep it internal. Happy to have a separate discussion about this at a later point though.